team.html

```diff
@@ -121,82 +109,90 @@ <h3>Controls</h3>
                     <div class="col-md-2 profile">
                         <img src="images/team/Tammy_Liu.png"/>
                         <h4 class="info">Tammy Liu</h4>
+                        <small>Advisor</small>
+                    </div>   
+                    <div class="col-md-2 profile">
+                        <img src="images/team/Jeff_Niu.png"/>
+                        <h4 class="info">Jeff Niu</h4>
                         <small>Team Lead</small>
-                    </div>                    
+                    </div>
                     <div class="col-md-2 profile">
                         <img src="images/team/Sadman_Kazi.jpg"/>
                         <h4 class="info">Sadman Kazi</h4>
-                        <small>Technical Lead</small>
                     </div>
                     <div class="col-md-2 profile">
                         <img src="images/team/William_Zou.jpg"/>
                         <h4 class="info">William Zou</h4>
                     </div>
                     <div class="col-md-2 profile">
-                        <img src="images/team/Jing_Li.jpg"/>
-                        <h4 class="info">Jing Li</h4>
-                    </div>
-                    <div class="col-md-2 profile">
-                        <img src="images/team/Jeff_Niu.png"/>
-                        <h4 class="info">Jeff Niu</h4>
-                    </div>
-                    <div class="col-md-2 profile">
-                        <h4 class="info">Ahmed Sabie</h4>
+                        <img src="images/team/Brett_Selby.jpg"/>
+                        <h4 class="info">Brett Selby</h4>
                     </div>
-                </div>
+                     <div class="col-md-2 profile">
+                        <img src="images/team/Beini_Fang.png"/>
+                        <h4 class="info">Beini Fang</h4>
+                    </div>>
```

Comment:
There seems to be an extra `>` character at the end of this. Also Beini is now the acting Team Lead.